### DISCUSSION TITLE
Can remove 'ContentTransition' from Expander in SimpleTheme? It looks strange.

### DISCUSSION THREAD

**Discussion by ZSYMAX** [AvaloniaUI/Avalonia#14218] [Q&A]

**Comment by timunie** [comment]
@ZSYMAX please ask things like that in Q&A discussion or on Telegram chat. You may need to override the Style for this. Check the original style file for what to modify. 

https://github.com/AvaloniaUI/Avalonia/blob/2a7b5568b7f61fe6df9c46a05bf1c9d474c9290e/src/Avalonia.Themes.Simple/Controls/Expander.xaml#L123-L127

**Comment by ZSYMAX** [comment]
now
![before](https://github.com/AvaloniaUI/Avalonia/assets/38749563/805bb289-4951-4b50-addd-0db38c815a40)

after, just remove the 'ContentTransition'
![after](https://github.com/AvaloniaUI/Avalonia/assets/38749563/d99ea2e6-6bf8-4144-98e7-b9398d981103)

**Reply by ZSYMAX** [reply]
window10  and  60hz screen
